src/IFrameNavigator.ts

```diff
@@ -294,191 +295,185 @@ export default class IFrameNavigator implements Navigator {
 
   protected async start(element: HTMLElement, entryUrl: URL): Promise<void> {
     element.innerHTML = template;
-    try {
```

Comment:
The main diff here is that I removed this try.  I might put it back because I'm not sure that I understand all of the specific problems that all of these things can throw, but I also as a general rule don't think that we should have such a big and generic try-catch